components/favourites.js

```diff
@@ -161,27 +161,39 @@ export class Favourites extends Component {
               <Paper padding="sm" className={contactUs}>
                 <Header2>{t("favourites.contact_us")}</Header2>
                 <p>
-                  <Link href={getLink(this.props.url, "map", "favourites")}>
+                  <HeaderButton
```

Comment:
Using the `useLink` prop for HeaderButton will preserve the Next `Link` that we're using here.